src/main/java/com/google/codeu/servlets/LoginServlet.java

```diff
@@ -38,8 +46,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // If the user is already logged in, redirect to their page
     if (userService.isUserLoggedIn()) {
-      String user = userService.getCurrentUser().getEmail();
-      response.sendRedirect("/users/" + user);
+      String userEmail = userService.getCurrentUser().getEmail();
+      createUserIfNew(userEmail);
```

Comment:
Since we're committing to making the user on login, I'm not sure if we necessarily need the two lines in the AboutMeServlet that previously made made the user in doPost.

Reply:
Good idea! I will see if I can use a getUser().setAboutMe() or something

Reply:
So, I think there's a bit of a problem with that one -- we have users stored using Google Cloud's userServiceFactory - and we can look these up with getCurrentUser(), but this returns Google's User class - this is not convertible to our custom User class. So, we end up having to create a new User object anyway, just like we already do. 

So, long story short, I think that when we create a new User in the aboutMe servlet, we are creating a local User object. Then, we add it to the Google Cloud userService, which automatically combines it as appropriate with any existing user, if that makes sense